apps/hash-api/src/graphql/resolvers/knowledge/page/update-page-actions.ts

```diff
@@ -37,11 +36,6 @@ export const createEntityWithPlaceholdersFn =
           draft.existingEntityId,
         ) as EntityId;
       }
-      if (draft.entityTypeId) {
-        draft.entityTypeId = placeholderResults.get(
-          draft.entityTypeId,
-        ) as VersionedUrl;
-      }
```

Comment:
The `placeholderResults` and placeholder map object was previously used to store placeholders for entity type IDs as well. Now that we can pass the exact entity type ID from BP, the placeholder map is solely for entity ids.